pycaret/time_series/forecasting/oop.py

```diff
@@ -3065,6 +3076,7 @@ def _plot_model(
         display_format: Optional[str] = None,
         data_kwargs: Optional[Dict] = None,
         fig_kwargs: Optional[Dict] = None,
+        display_kwargs: Optional[Dict] = None,
```

Comment:
All `plotly` arguments go into `fig_kwargs`. Can we add the `display_kwargs` there as well instead of introducing another argument?

Reply:
I assume this is certainly possible! For example; `display_kwargs` can reside under the `display_kwargs` key within `fig_kwargs` etc ..

It seems we have two options; (1) put show dash on the same level as `display_kwargs` or let it reside within `display_kwargs`. What would you prefer?
```python
# option 1
fig_kwargs = {
  ...,
  "display_kwargs": { ... },  # the new name for the display kwargs
  "show_dash": { ... }, 
}

# option 2
fig_kwargs = {
  ...,
  "display_kwargs": {
     ...,
     "show_dash": { ... }
   } 
}
```

Reply:
Hi @jonasvdd , Can you rename `display_kwargs` to `resampler_kwargs`. Then you can go with Option 2 since all the keys will be related to the resampler.

Reply:
As for now, I will implement it as suggested by @ngupta23 in [81ee7c9](https://github.com/pycaret/pycaret/pull/2671/commits/81ee7c980f960f8a6633ab5191a122f0ea74c81e)